src/wallet/gtest/test_wallet.cpp

```diff
@@ -1147,7 +1172,8 @@ TEST(WalletTests, SpentSaplingNoteIsFromMe) {
 }
 
 TEST(WalletTests, CachedWitnessesEmptyChain) {
-    TestWallet wallet;
+    SelectParams(CBaseChainParams::REGTEST);
```

Comment:
This seems like it is probably relying on the order of tests (to avoid this, there are likely to be other tests that need to explicitly set REGTEST, or else we should automatically set some specific default). Please open a ticket.